pkgs/os-specific/linux/libselinux/default.nix

```diff
@@ -43,6 +43,10 @@ stdenv.mkDerivation rec {
     "LIBSEPOLA=${lib.getLib libsepol}/lib/libsepol.a"
   ];
 
+  prePatch = lib.optional stdenv.hostPlatform.isMusl ''
+    sed -i 's,#include <unistd.h>,,g' src/procattr.c
```

Comment:
Please use subsituteInPlace.

Reply:
done in 07b190c8bdf883ae899f53d56d7f6bfea6a12cd8